From 7d6ffa34ee351fb25250b2cd25e1085acd1f2731 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 14 May 2009 20:56:58 -0700
Subject: Staging: cpc-usb: fix some build problems in the driver

It will now build properly on the latest kernel tree.

Cc: Sebastian Haas <haas@ems-wuensche.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/cpc-usb/cpc-usb_drv.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

--- a/drivers/staging/cpc-usb/cpc-usb_drv.c
+++ b/drivers/staging/cpc-usb/cpc-usb_drv.c
@@ -49,7 +49,7 @@
 /* Version Information */
 #define DRIVER_AUTHOR  "Sebastian Haas <haas@ems-wuensche.com>"
 #define DRIVER_DESC    "CPC-USB Driver for Linux Kernel 2.6"
-#define DRIVER_VERSION CPC_DRIVER_VERSION " (CDKL v" CDKL_VERSION ")"
+#define DRIVER_VERSION CPC_DRIVER_VERSION
 
 MODULE_AUTHOR(DRIVER_AUTHOR);
 MODULE_DESCRIPTION(DRIVER_DESC);
@@ -1198,7 +1198,6 @@ static int __init CPCUsb_Init(void)
 	if (!procDir) {
 		err("Could not create proc entry");
 	} else {
-		procDir->owner = THIS_MODULE;
 		procEntry = create_proc_read_entry("info", 0444, procDir,
 						   cpcusb_proc_read_info,
 						   NULL);
@@ -1206,8 +1205,6 @@ static int __init CPCUsb_Init(void)
 			err("Could not create proc entry %s", CPC_USB_PROC_DIR "/info");
 			remove_proc_entry(CPC_USB_PROC_DIR, NULL);
 			procDir = NULL;
-		} else {
-			procEntry->owner = THIS_MODULE;
 		}
 	}
 
